roles/nf-core/defaults/main.yml

```diff
@@ -1,17 +1,24 @@
 nextflow_java: "/sw/comp/java/x86_64/sun_jdk1.8.0_151"
-nextflow_version_tag: "v21.04.1"
+nextflow_version_tag: "v21.04.3"
 nextflow_download_url: "https://github.com/nextflow-io/nextflow/releases/download/{{ nextflow_version_tag }}/nextflow"
-nf_core_env: "/lupus/ngi/irma3/nf-core-env"
+nf_core_env: "/lupus/ngi/irma3/nf-core-env-v2"
+nf_core_tools_version: 2.1
+biocontainers_dirname: "biocontainers"
 pipelines:
   - name: methylseq
-    release: 1.6
-  - name: ampliseq
-    release: 1.1.2
+    release: 1.6.1
   - name: atacseq
-    release: 1.1.0
-  - name: viralrecon
-    release: 1.1.0
+    release: 1.2.1
   - name: rnafusion
     release: 1.2.0
   - name: nanoseq
     release: 1.1.0
+  - name: ampliseq
+    release: 2.0.0
+    modified_singularity_cachedir: "{{ biocontainers_dirname }}"
```

Comment:
Instead of repeating `biocontainers_dirname`, we could maybe have a flag marking whether the pipeline isDSL2 or not, since thats what it seems to be used to check in the site config, if I'm not wrong?

Reply:
I actually thought it would be more flexible if you can set the singularity cachedir to the directory name that you want, e.g. if you want to have a separate folder for a few pipelines but another for other ones? So what the config does to achieve this is that it sets this modified directory name (if you do not want to use the pipeline name) as the singularity cachedir, so that it can be variable. What do you say? Do you think there are any advantage of making it variable or should we substitute it to a boolean flag?

Reply:
I'll change it to a flag

Reply:
hmm, started changing the solution into a boolean flag but realised that the way I solved it in the beginning made this part pretty tidy: https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/0fadf3ecbf4a9a483615741149cd567b48b5561d/roles/nf-core/tasks/main.yml#L53 

compared to changing the flag `modified_singularity_cachedir` to a boolean and setting the same line to
```
container_dir: "{{ biocontainers_dirname if item.modified_singularity_cachedir is defined and item.modified_singularity_cachedir == True else item.name}}"
```
However I'm still open to a nice solution if you have any ideas :)

Reply:
It should be fine to have it as it is then.